src/coreclr/vm/arm64/asmhelpers.S

```diff
@@ -124,7 +124,6 @@ LEAF_ENTRY HelperMethodFrameRestoreState, _TEXT
 LOCAL_LABEL(Done):
```

Comment:
Isn't the whole `HelperMethodFrameRestoreState` going to be dead code?

Reply:
PERMIT_HELPER_METHOD_FRAME_ removed all the way up to unwindLazyState, which was my objective https://github.com/dotnet/runtime/pull/115134#issuecomment-2849731870 😅